antiplag/serializers.py

```diff
@@ -64,4 +64,4 @@ class DocumentResultSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Document
-        fields = ("name", "result")
+        fields = ("name", "result", "text")
```

Comment:
Is this part of task's scope?

Reply:
Nop, it's in the scope of this task: https://p3k-team.atlassian.net/browse/BE-80?atlOrigin=eyJpIjoiMjkwNGZmZTU2MzNhNDA2YmIzNmEyMTA3YjlmMjc4YjAiLCJwIjoiaiJ9

Task's scope is only to provide document content which is 1 word, therefore I did it as a part of task BE-88.

Reply:
Pls move it to its own branch. Even though it's such a small task.

Reply:
Because this file is moved to its own branch. It's not here anymore.